### PR TITLE
Fetch drink categories api

### LINKED ISSUE #40: [Js] fetch Drink categories from api (categoriesDrink page)

### LINKED ISSUE #42: [Js] addEventListener click to image categories (categoriesDrink page)
addEventListener click to image categories and go to typeDrink page and store the name Category in local Storage

### PR DESCRIPTION
**Pull request by MohammedDahalan**

### CODE REVIEW

**Review by IsraaHamdi — CHANGES_REQUESTED** [review]
delete element in html file

File: js/categoryDrink.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+let catDrinkName=[]
+fetch('https://www.thecocktaildb.com/api/json/v1/1/list.php?c=list')
+.then(res =>res.json())
+.then(
+    data =>  {
+        let categories=data.drinks
+        categories.forEach(ele => {
+            catDrinkName.push(`${ele.strCategory}`)     
+    })
+    console.log(catDrinkName)
```
Comment (yaqootturman) [inline js/categoryDrink.js]: delete console

File: js/categoryDrink.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    localStorage.setItem('catDrinkName',JSON.stringify(catDrinkName))
+ }   
+)
+.catch(err => alert(err))
+
+let catDrinkNameLocal=JSON.parse(localStorage.getItem("catDrinkName"))
+    console.log(catDrinkNameLocal)
+    console.log(catDrinkNameLocal.length)
```
Comment (yaqootturman) [inline js/categoryDrink.js]: delete console

File: js/categoryDrink.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+for( let i=0;i<catDrinkNameLocal.length;i++) {
+           console.log(catDrinkNameLocal.length)
```
Comment (yaqootturman) [inline js/categoryDrink.js]: same

File: js/categoryDrink.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+let catDrink=document.getElementsByClassName('categories-drink')
+let catDrinkName=[]
+fetch('https://www.thecocktaildb.com/api/json/v1/1/list.php?c=list')
```
Comment (yaqootturman) [inline js/categoryDrink.js:5]: Same for Food categories, try to ask if data is in local storage or not, and if it is in local storage get them from it, not from the API

**Review by yaqootturman — CHANGES_REQUESTED** [review]
change files names

File: js/categoryDrink.js (hunk 1 above)
Comment (IsraaHamdi) [reply js/categoryDrink.js]: done

File: js/categoryDrink.js (hunk 2 above)
Comment (IsraaHamdi) [reply js/categoryDrink.js]: done

File: js/categoryDrink.js (hunk 3 above)
Comment (IsraaHamdi) [reply js/categoryDrink.js]: done

File: js/categoryDrink.js (hunk 4 above)
Comment (IsraaHamdi) [reply js/categoryDrink.js:5]: done

**Review by farahalzard — APPROVED** [review]

**Review by IsraaHamdi — APPROVED** [review]

**Review by muhannaAtalla20 — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**